lib/msf/core/optional_session.rb

```diff
@@ -7,10 +7,20 @@
 module Msf::OptionalSession
   include Msf::SessionCompatibility
 
+  REQUIRED_FEATURE_FLAGS = [
+    Msf::FeatureManager::SMB_SESSION_TYPE,
+    Msf::FeatureManager::POSTGRESQL_SESSION_TYPE
+  ].freeze
+
+  def supported?
```

Comment:
I believe you'll want to namespace this properly, otherwise you might break existing third-party modules that have a `supported?` method present